driver/ppm_flag_helpers.h

```diff
@@ -1591,6 +1591,16 @@ static __always_inline u32 mlockall_flags_to_scap(unsigned long flags)
 	return res;
 }
 
+static __always_inline u32 mlock2_flags_to_scap(unsigned flags)
```

Comment:
```suggestion
static __always_inline u32 mlock2_flags_to_scap(unsigned long flags)
```
I think that `unsigned long` here is safer :)